### ISSUE TITLE
Missing jinja2 dependency

### ISSUE THREAD

**Issue by timtroendle** [conda-forge/calliope-feedstock#16]
Issue:

The current `calliope` build seems to lack a dependency on `jinja2`. Steps to reproduce:

```bash
conda create -n testtest calliope -c conda-forge --override-channels
conda activate testtest
ipython

In [1]: import calliope
[...]
ModuleNotFoundError: No module named 'jinja2'
```

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# Name                    Version                   Build  Channel
appdirs                   1.4.3                      py_1    conda-forge
appnope                   0.1.0                 py36_1000    conda-forge
asn1crypto                0.24.0                py36_1003    conda-forge
atomicwrites              1.2.1                      py_0    conda-forge
attrs                     18.2.0                     py_0    conda-forge
backcall                  0.1.0                      py_0    conda-forge
blas                      1.1                    openblas    conda-forge
bottleneck                1.2.1            py36h7eb728f_1    conda-forge
bzip2                     1.0.6                         1    conda-forge
ca-certificates           2018.10.15           ha4d7672_0    conda-forge
calliope                  0.6.3                      py_2    conda-forge
certifi                   2018.10.15            py36_1000    conda-forge
cffi                      1.11.5           py36h5e8e0c9_1    conda-forge
chardet                   3.0.4                 py36_1003    conda-forge
click                     6.7                        py_1    conda-forge
cryptography              2.3.1            py36hdffb7b8_0    conda-forge
cryptography-vectors      2.3.1                 py36_1000    conda-forge
curl                      7.62.0               h74213dd_0    conda-forge
decorator                 4.3.0                      py_0    conda-forge
glpk                      4.65                 h16a7912_1    conda-forge
gmp                       6.1.2                hfc679d8_0    conda-forge
hdf4                      4.2.13               h951d187_2    conda-forge
hdf5                      1.8.18                        3    conda-forge
idna                      2.7                   py36_1002    conda-forge
ipykernel                 5.1.0           py36h24bf2e0_1001    conda-forge
ipython                   7.1.1           py36h24bf2e0_1000    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
jedi                      0.13.1                py36_1000    conda-forge
jpeg                      9c                   h470a237_1    conda-forge
jsonschema                3.0.0a3               py36_1000    conda-forge
jupyter_client            5.2.3                      py_1    conda-forge
jupyter_core              4.4.0                      py_0    conda-forge
krb5                      1.16.2               hbb41f41_0    conda-forge
libcurl                   7.62.0               hbdb9355_0    conda-forge
libedit                   3.1.20170329         haf1bffa_1    conda-forge
libffi                    3.2.1                hfc679d8_5    conda-forge
libgfortran               3.0.0                         1    conda-forge
libnetcdf                 4.4.1.1                       9    conda-forge
libsodium                 1.0.16               h470a237_1    conda-forge
libssh2                   1.8.0                h5b517e9_3    conda-forge
more-itertools            4.3.0                 py36_1000    conda-forge
natsort                   5.5.0                      py_0    conda-forge
nbformat                  4.4.0                      py_1    conda-forge
ncurses                   6.1                  hfc679d8_1    conda-forge
netcdf4                   1.3.1                    py36_0    conda-forge
nose                      1.3.7                 py36_1002    conda-forge
numexpr                   2.6.8            py36hf8a1672_0    conda-forge
numpy                     1.15.4          py36_blas_openblashb06ca3d_0  [blas_openblas]  conda-forge
openblas                  0.3.3                ha44fe06_1    conda-forge
openssl                   1.0.2p               h470a237_1    conda-forge
pandas                    0.22.0                   py36_1    conda-forge
parso                     0.3.1                      py_0    conda-forge
pexpect                   4.6.0                 py36_1000    conda-forge
pickleshare               0.7.5                 py36_1000    conda-forge
pip                       18.1                  py36_1000    conda-forge
plotly                    3.4.1                      py_0    conda-forge
pluggy                    0.8.0                      py_0    conda-forge
ply                       3.11                       py_1    conda-forge
prompt_toolkit            2.0.7                      py_0    conda-forge
ptyprocess                0.6.0                 py36_1000    conda-forge
py                        1.7.0                      py_0    conda-forge
pycodestyle               2.4.0                      py_1    conda-forge
pycparser                 2.19                       py_0    conda-forge
pygments                  2.2.0                      py_1    conda-forge
pyomo                     5.4.3                    py36_0    conda-forge
pyopenssl                 18.0.0                py36_1000    conda-forge
pyrsistent                0.14.6           py36h470a237_0    conda-forge
pysocks                   1.6.8                 py36_1002    conda-forge
pytest                    4.0.0                 py36_1000    conda-forge
python                    3.6.6                h5001a0f_0    conda-forge
python-dateutil           2.7.5                      py_0    conda-forge
pytz                      2018.7                     py_0    conda-forge
pyutilib                  5.6.3                    py36_0    conda-forge
pyzmq                     17.1.2           py36hae99301_1    conda-forge
readline                  7.0                  haf1bffa_1    conda-forge
requests                  2.20.1                py36_1000    conda-forge
retrying                  1.3.3                      py_2    conda-forge
ruamel.yaml               0.15.78          py36h470a237_0    conda-forge
scikit-learn              0.20.0          py36_blas_openblash00c3548_201  [blas_openblas]  conda-forge
scipy                     1.1.0           py36_blas_openblashb06ca3d_202  [blas_openblas]  conda-forge
setuptools                40.6.2                   py36_0    conda-forge
six                       1.11.0                py36_1001    conda-forge
sqlite                    3.25.3               hb1c47c0_0    conda-forge
tk                        8.6.9                ha92aebf_0    conda-forge
tornado                   5.1.1            py36h470a237_0    conda-forge
traitlets                 4.3.2                 py36_1000    conda-forge
urllib3                   1.23                  py36_1001    conda-forge
wcwidth                   0.1.7                      py_1    conda-forge
wheel                     0.32.3                   py36_0    conda-forge
xarray                    0.10.9                py36_1000    conda-forge
xz                        5.2.4                h470a237_1    conda-forge
zeromq                    4.2.5                hfc679d8_6    conda-forge
zlib                      1.2.11               h470a237_3    conda-forge
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info
     active environment : testtest
    active env location : /Users/trtim/miniconda3/envs/testtest
            shell level : 2
       user config file : /Users/trtim/.condarc
 populated config files : /Users/trtim/.condarc
          conda version : 4.5.11
    conda-build version : 3.16.1
         python version : 3.7.0.final.0
       base environment : /Users/trtim/miniconda3  (writable)
           channel URLs : https://conda.anaconda.org/timtroendle/osx-64
                          https://conda.anaconda.org/timtroendle/noarch
                          https://conda.anaconda.org/conda-forge/osx-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/osx-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/pro/osx-64
                          https://repo.anaconda.com/pkgs/pro/noarch
          package cache : /Users/trtim/miniconda3/pkgs
                          /Users/trtim/.conda/pkgs
       envs directories : /Users/trtim/miniconda3/envs
                          /Users/trtim/.conda/envs
               platform : osx-64
             user-agent : conda/4.5.11 requests/2.19.1 CPython/3.7.0 Darwin/18.2.0 OSX/10.14.1
                UID:GID : 502:20
             netrc file : None
           offline mode : False
```
</details>

**Comment by sjpfenninger** [comment]
Not sure why that wasn't picked up, as it is supposed to try importing calliope after building the package -- anyway that is fixed as of 18ff737 and should be online in 1-2 hours once the new packages are built.

**Comment by timtroendle** [comment]
Maybe it used to be a transitive dependency and the direct dependency changed it the meantime? Just guessing.
